src/web/gwc/src/main/java/org/geoserver/gwc/web/GWCServiceDescriptionProvider.java

```diff
@@ -25,7 +25,7 @@
 public class GWCServiceDescriptionProvider extends ServiceDescriptionProvider {
 
     /** Service used to cross-link between service description and service link description. */
-    public static final String SERVICE = "gwc";
+    public static final String SERVICE = "wmts";
```

Comment:
Seeking consistency with ``GwcServiceProxy`` in use of ``wmts`` as service type.

The string ``WMTS`` is used by GWCResourceServiceVoter to avoid showing this as a service enablement.